gremlin-python/src/main/python/gremlin_python/process/graph_traversal.py

```diff
@@ -1424,6 +1428,10 @@ def ltrim(cls, *args):
     def l_trim(cls, *args):
         return cls.graph_traversal(None, None, Bytecode()).l_trim(*args)
 
+    @classmethod
```

Comment:
Looks like there's an issue with merge.
Duplicate `l_trim` and `r_trim` function definition (4 times in total).

Reply:
thanks for noticing that.